installer/packer/scripts/harbor/harbor.cfg

```diff
@@ -0,0 +1,113 @@
+## Configuration file of Harbor
+
+#The IP address or hostname to access admin UI and registry service.
+#DO NOT use localhost or 127.0.0.1, because Harbor needs to be accessed by external clients.
+hostname = reg.mydomain.com
+
+#The protocol for accessing the UI and token/notification service, by default it is http.
```

Comment:
This is the default configuration file, it is managed by the `configure_harbor.sh` script and is forced to be https only, the cipher suite is determined by the nginx config and provided by the harbor team https://github.com/vmware/harbor/blob/master/make/common/templates/nginx/nginx.https.conf#L34-L37